Automattic/wp-calypso
# Domains: Consolidate TLD filters into single bar

This change consolidates TLD filters into one bar in the domain search experience, like so:

<img width="979" alt="wordpress_com" src="https://user-images.githubusercontent.com/4044428/39455184-67a43582-4c9c-11e8-812d-17ccc279d968.png">

This change enables the TLD filters bar in `development`, `horizon`, and `wpcalypso`.

# Testing instructions
1. Spin up this branch locally.
2. Navigate to `/start/domains` and enter a query.
3. Ensure that the TLD filters bar appears.
4. Ensure that clicking a TLD filter button works as expected.
